oprf/oprf.go

```diff
@@ -62,13 +62,13 @@ type Token struct {
 
 // Evaluation corresponds to the evaluation over a token.
 type Evaluation struct {
-	element []byte
+	Element []byte
 }
 
 // KeyPair is an struct containing a public and private key.
 type KeyPair struct {
-	pubK  *group.Element
-	PrivK *group.Scalar
+	PublicKey  *group.Element
```

Comment:
It was previously argued that the Public Key did not need to be exported. Why is it exported now?

Reply:
Hmm, was it? I don't remember why. I can unexport it.